boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -19,6 +19,25 @@
 import argparse
 import string
 
+BUNDLE_PERMITTED_RESOURCES = (
```

Comment:
Can we include a comment linking to the source for this?

https://github.com/operator-framework/operator-registry/blob/release-4.7/pkg/lib/bundle/supported_resources.go#L3-L20

Reply:
Also useful in the comment would be to note that we [pinned the registry version to 4.7.0](https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/csv-generate/catalog-build.sh#L57), so upgrading to support new kinds would entail bumping that version as well as this list.